### ISSUE TITLE
Aeschynomene sesban (Q21870070) has a type that isn't a type

### ISSUE THREAD

**Issue by rdmpage** [rdmpage/alec#11]
[Q21870070](https://alec-demo.herokuapp.com/Q21870070) has Aeschynomene sesban Jacq. listed as a type, whereas this is incorrect (see https://www.wikidata.org/wiki/User_talk:Rdmpage#synonyms_abd_their_references ). Why does a taxon appear as a type for a species?

**Comment by rdmpage** [comment]
OK, the query [Try it](https://w.wiki/3RsU) is broken as it should test whether we are looking at a nomenclatural type, but it isn't bound to anything.

```
SELECT * WHERE {
  VALUES ?taxon { wd:Q21870070 }

# find thing which has a role "of or for" this taxon  
  ?statement pq:P642 ?taxon .
  ?statement ps:P2868 ?item_type .
  ?item p:P2868 ?statement .
 
  # ensure thing is a nomenclatural type
  ?type_of_type wdt:P279 wd:Q3707858 .
  
  # labels
  ?item_type rdfs:label ?type_of_type_label .
  ?item rdfs:label ?name .
  
  FILTER(LANG(?type_of_type_label) = "en")
  FILTER(LANG(?name) = "en")	
}
```

**Comment by rdmpage** [comment]
Fixed with b6ac7492bacec1cf8c0d76737283f65e0b7e27b7
